src/Reader/Entry/Rss.php

```diff
@@ -275,7 +275,7 @@ public function getDescription()
     /**
      * Get the entry enclosure
      *
-     * @return string
+     * @return object
```

Comment:
This should be reverted because it is unrelated to this pull request. It can be added to a separated bug fix release.